catalogue/webapp/pages/workv2.js

```diff
@@ -52,7 +52,7 @@ export const WorkPage = ({
         imageAltText={null}>
```

Comment:
As above, `<PageLayout>` is now included in `<ErrorPage>` so we can remove it from here